apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -115,15 +115,15 @@ const ExpandedCurriculumCatalogCard = ({
                       {i18n.availableResources()}
                     </Heading4>
                     <hr className={style.thickDivider} />
-                    <TextLink text={i18n.lessonPlans()} href="#" />
+                    <BodyTwoText>{i18n.lessonPlans()} </BodyTwoText>
```

Comment:
Is the plan then in the future to wrap these in `<a>` tags? If so, we want to make sure the text will still highlight and behave like links

Reply:
It looks like Molly gave two options:
- The list items under Available Resources should be formatted as body text (Gotham 4r, Light/Black [#292F36]) OR as a link (Gotham 5r, Light/Secondary/500 [#8C52BA], underlined). The current text weight is too heavy and when I hover, I see the link formatting.

I went with the first option as Dani told me for now that the available resources wont be links but instead just text that shows if the course has that resource available.